packages/@aws-cdk/aws-ecr-assets/test/image-asset.test.ts

```diff
@@ -405,6 +405,30 @@ describe('image asset', () => {
     expect(asset1.assetHash).toEqual('13248c55633f3b198a628bb2ea4663cb5226f8b2801051bd0c725950266fd590');
     expect(asset2.assetHash).toEqual('b78978ca702a8eccd37804ce31d76cd83a695b557dbf95aeb109332ee8b1fd32');
   });
+
+  describe('stack synthesizers', () => { //synthesizedTag and assetHash
```

Comment:
```suggestion
  describe('imageTag is correct for different stack synthesizers', () => {
```

the comment doesn't mean much to me here. I'm not sure it's necessary at all. But if you're going to add comments, they should be far more descriptive. Rather, you should use the comment inside the 'describe' block to effectively explain what you are testing.

Reply:
comment was not necessary